deploy/salt/roots/salt/memex-explorer.sls

```diff
@@ -14,12 +14,9 @@ local_settings_path:
      - name: LOCAL_SETTINGS_PATH
      - value: /vagrant/source/memex/local_settings.py
 
-
-# TODO: The create_apps_Tika_ES_Kibana is not idempotent.  This needs
-# to be fixed or the command needs to be protected so that it only
-# executes when the applications need to be created.
-# As a workaround, the SQL database is currently reset on each provision.
-
+# We originally reset the database on each reprovision because of a database-modifying function that was not idempotent.
+# This may not be necessary.
+# @ahmadia, is there any other reason for it?
```

Comment:
@amfarrell - this has been fixed, let's stop deleting the database on reprovision.  It would also probably be good to move the database to /home/vagrant.